spec/services/course_exports/prepare_teams_export_service_spec.rb

```diff
@@ -241,10 +241,7 @@ def report_link_formula(student)
             team_2.name,
             nil,
             '1/2',
-            {
-              'value' => submission_grading(team_2_reviewed_submission),
-              'style' => 'failing-grade'
-            },
+            'x',
```

Comment:
From the last discussion with the team, we decided to show all submissions from a student in the export including the deleted submissions.